metadata/society/ems_response_areas.md

```diff
@@ -26,11 +30,11 @@ The dataset was created to provide response recommendations for the Next-Generat
 
```

Comment:
Yes, it does. The main datasets that are tied to NG911 directly are Law, Fire, and EMS.